AirCasting/SessionViews/ShareViewModal.swift

```diff
@@ -49,26 +57,35 @@ struct ShareViewModal: View {
     
     private var shareButton: some View {
         Button("Share link") {
-            presentationMode.wrappedValue.dismiss()
+            sheet.toggle()
         }.buttonStyle(BlueButtonStyle())
-        .padding(.bottom)
+            .padding(.bottom)
     }
     
     private var descriptionMail: some View {
         Text(Strings.sessionShare.emailDescription)
-            .font(.subheadline)
+            .font(Font.muli(size: 12))
             .foregroundColor(.aircastingGray)
     }
     
     private var oKButton: some View {
         Button("Share file") {
-            presentationMode.wrappedValue.dismiss()
+            if MFMailComposeViewController.canSendMail() {
+                isShowingMailView.toggle()
+            } else {
+                showingAlert = !isShowingMailView
+                print("no email app")
```

Comment:
let's make our prints more descriptive like "Not showing mail view (cannot send email from this device)" or smth. Also - can we use our `Log.info("asdasd")` insetead here?